resume-buddy/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -112,4 +107,56 @@ private boolean collides(UUID id) {
 
     return results.countEntities(FetchOptions.Builder.withDefaults()) != 0;
   }
+
+  /* checks if given user has a match */
```

Comment:
"Checks if the given user of the given userType has a match of the opposite userType."